package/sbp_anpp_bridge/src/sbp_anpp_bridge.c

```diff
@@ -51,30 +51,44 @@
 #define SBP_MSG_ODOMETRY_FLAGS_SRC2 (0x08u)
 #define SBP_MSG_ODOMETRY_FLAGS_SRC3 (0x0Cu)
 
-#define INVALID_TOW 0xffffffff
-#define TIME_SOURCE_MASK 0x07 /* Bits 0-2 */
-#define NO_TIME 0
-#define MS_IN_WEEK 7 * 24 * 60 * 60 * 1000
+#define INVALID_TOW ((unsigned int)0xffffffff)
+#define TIME_SOURCE_MASK ((unsigned short)0x07) /* Bits 0-2 */
+#define NO_TIME ((unsigned int)0)
+#define MS_IN_WEEK ((double)7 * 24 * 60 * 60 * 1000)
 
 bool anpp_debug = false;
-unsigned int last_gpstime_tow = INVALID_TOW;
-struct timespec last_gps_systime;
-int odo_time_offset_ms = 0;
+
+typedef struct anpp_bridge_ctx_s {
+  unsigned int last_gpstime_tow;
+  struct timespec last_gps_systime;
+  int odo_time_offset_ms;
+  pk_endpoint_t *anpp_sub;
+} anpp_bridge_ctx_t;
```

Comment:
you can use struct initializers here instead of using an explicit init function: